build.sbt

```diff
@@ -170,6 +170,13 @@ lazy val state    = crossProject.dependsOn(core)
   .configureCross(monocleCrossSettings)
   .settings(libraryDependencies ++= Seq(scalaz.value))
 
+lazy val readerJVM = reader.jvm
```

Comment:
I agree "state" is a nice name. `ReaderGetterSyntax` has been moved to *state*, and all references to *reader* have been removed.